library-best-practices/glossary.md

```diff
@@ -37,9 +38,9 @@ Java Dependency Glossary
 <a name="linkage-conflict"></a>
 <a name="static-linkage-conflict"></a>
 <a name="dynamic-linkage-conflict"></a>
-- **Linkage conflict**: a linkage error when the signature, return type,
-  modifiers, or throws declaration of a non-private method, field, or class
-  in a dependency has changed (or removed) in an incompatible way between
+- **Linkage conflict**: a linkage error when the name, return type,
+  modifiers, arguments, or throws clause of a non-private method, field, or class
```

Comment:
I found a change to `throws` clause does not break compatibility. Shall we remove this part?

> Changes to the throws clause of methods or constructors do not break compatibility with pre-existing binaries; these clauses are checked only at compile time.
https://docs.oracle.com/javase/specs/jls/se8/html/jls-13.html#jls-13.4.21